### DISCUSSION TITLE
[solved] Engrave and cut in one file?

### DISCUSSION THREAD

**Discussion by wega-svg** [arkypita/LaserGRBL#2091] [Q&A]
Hello,
can I engrave and cut in one file?
If yes, how does it work?

Sincerely Werner

**Comment by Tribo-2dot0** [comment]
Use a função Append File...
Duas maneiras:
https://www.youtube.com/watch?v=LTU_xWh286I

https://www.youtube.com/watch?v=k7LZk5Qr7zI

**Reply by wega-svg** [reply]
Thanks for these suggestions. That helped a lot.
With best regards, Werner

**Comment by philnorv** [comment]
Some materials need cutting with multiple passes. There is no feature that I can see to set a different number of passes for an appended file from the number of passes for the first file. The workaround is to append the file used for the cut lines multiple times one after another.

**Reply by Tribo-2dot0** [reply]
Uma outra alternativa é, em Inkscape, selecionar as linhas de corte e duplica-las (Ctrl+D) o número de vezes necessário ao número de passes; as linhas ficarão sobrepostas.
